app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -163,6 +155,15 @@ def textual_health_state
     {:label => _("Health State"), :value => @record.health_state}
   end
 
+  def textual_network_devices
+    hardware_nics_count = @record.hardware.nics.count
```

Comment:
`@record.guest_devices` would be the same as `@record.hardware.nics`, right?

Do you want to use `@record.guest_devices.count` here in order to be consistent with the nested display list? 
i.e. both the link and the actual list rely on the PhysicalServer/GuestDevice association

Reply:
@AparnaKarve `@record.hardware.nics` is different because it returns guest devices that have a _device_type_ of 'ethernet' which is what we want here. `@record.guest_devices` would return all guest devices including storage adapters, ports, etc. which we don't want included in the count.

Reply:
Ok, thanks for that clarification.
`@record.hardware.nics` makes sense to get the count for the 'ethernet' guest devices